app/parse.py

```diff
@@ -8,5 +11,26 @@
     duration: str
 
 
+def parse_single_course(course_element: Tag) -> Course:
+    name = course_element.select_one("h3").text.strip()
+    short_description = course_element.select_one(
+        ".typography_landingTextMain__Rc8BD"
+    ).text.strip()
+    duration = course_element.select(
+        ".typography_landingTextMain__Rc8BD span"
+    )[-1].text.strip()
+
+    return Course(name, short_description, duration)
+
+
 def get_all_courses() -> list[Course]:
-    pass
+    content = requests.get("https://mate.academy/")
```

Comment:
Consider adding error handling for the HTTP request to handle potential network issues or invalid responses more gracefully.